l10n_be_cooperator_portal_national_number/controllers/portal.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from unittest import mock
+
+from odoo import _
+from odoo.http import request, route
+
+from odoo.addons.account.controllers.portal import PortalAccount
+
+
+class CooperatorPortal(PortalAccount):
+    def __init__(self, *args, **kwargs):
+        super().__init__()
```

Comment:
why is this called without arguments? please add a comment or remove the line.

Reply:
was copied from `cooperator_portal`, fixed it here and there